node/src/database/db_migrations.rs

```diff
@@ -470,6 +500,193 @@ impl Migrate_5_to_6 {
     }
 }
 
+#[allow(non_camel_case_types)]
+struct Migrate_6_to_7;
+
+#[allow(non_camel_case_types)]
+struct Migrate_6_to_7_carrier<'a> {
+    utils: Box<dyn MigDeclarationUtilities + 'a>,
+    statements: Vec<Box<dyn StatementObject>>,
+}
+
+impl DatabaseMigration for Migrate_6_to_7 {
+    fn migrate<'a>(
+        &self,
+        declaration_utils: Box<dyn MigDeclarationUtilities + 'a>,
+    ) -> rusqlite::Result<()> {
+        let mut migration_carrier = Migrate_6_to_7_carrier::new(declaration_utils);
+        migration_carrier.retype_table(
+            "payable",
+            "balance",
+            "wallet_address text primary key,
+                              balance_high_b integer not null,
+                              balance_low_b integer not null,
+                              last_paid_timestamp integer not null,
+                              pending_payable_rowid integer null",
+        )?;
+        migration_carrier.retype_table(
+            "receivable",
+            "balance",
+            "wallet_address text primary key,
+                             balance_high_b integer not null,
+                             balance_low_b integer not null,
+                             last_received_timestamp integer not null",
+        )?;
+        migration_carrier.retype_table(
+            "pending_payable",
+            "amount",
+            "transaction_hash text not null,
+                              amount_high_b integer not null,
+                              amount_low_b integer not null,
+                              payable_timestamp integer not null,
+                              attempt integer not null,
+                              process_error text null",
+        )?;
+        migration_carrier.statements.push(Box::new(format!(
+            "update config set value = '{}' where name = 'rate_pack'",
```

Comment:
Possibly might you want to set the new rate pack based on the old rate pack, rather than back to default?